index.html

```diff
@@ -4,164 +4,188 @@
               </div>
             </div>
-            <div class="">
-              <div class="">
-
+            <div class="grid-item">
+              <div class="grid-img">
+                <img src="https://www.revzilla.com/product_images/0351/5247/feuling_oe_hydraulic_cam_chain_tensioner_conversion_kit_for_harley20012006_750x750.jpg" alt="" class="w-95">
               </div>
-              <div class="">
-
+              <div class="grid-content">
+                <h6 class="h5">Feuling OE+ Hydraulic Cam Chain Tensioner Conversion Kit For Harley 2001-2006</h6>
+                <p class="text-lightgray h6"><span>Base Price:</span>$699.95</p>
               </div>
             </div>
-            <div class="">
-              <div class="">
-
+            <div class="grid-item">
+              <div class="grid-img">
+                <img src="https://www.revzilla.com/product_images/0224/5383/ss_kit_hsu959906_750x750.jpg" alt="" class="w-95">
               </div>
-              <div class="">
-
+              <div class="grid-content">
+                <h6 class="h5">S&S 95" Hot Set Up Kit For 88" Harley Twin Cam 1999-2006</h6>
+                <p class="text-lightgray h6"><span>Base Price:</span>$3,412.95</p>
               </div>
             </div>
           </div>
         </div>
       </div>
     </main>
-    <footer>
-      <div class="">
-        <div class="">
-
+    <footer class="mb-5">
+      <div class="container d-flex js-cn-sp-bt mt-5">
+        <div class="w-15">
+          <img src="./assets/image/logo.jpg" alt="" class="w-95">
         </div>
-        <div class="">
-
+        <div class="w-25">
+          <ul class="text-decoration-none style-none d-flex js-cn-sp-bt">
+            <li class="text-lightblue"><i class="fab fa-facebook-f"></i></li>
+            <li class="text-lightblue"><i class="fab fa-twitter"></i></li>
+            <li class="text-lightblue"><i class="fab fa-instagram"></i></li>
+            <li class="text-lightblue"><i class="fab fa-tumblr"></i></li>
+            <li class="text-lightblue"><i class="fas fa-envelope"></i></li>
+          </ul>
         </div>
       </div>
-      <div class="">
-
+      <div class="text-center">
+        <p class="text-white">(c) On Road.com / All right to whoever </p>
       </div>
     </footer>
-    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
-    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
```

Comment:
remove bootstrap file from this project.

Reply:
again, you are light on the backdated file.